src/Storage/Storage.Management/help/New-AzRmStorageShare.md

```diff
@@ -84,18 +88,18 @@ myshare                                                                       20
 This command creates a Storage file share snapshot of an existing base file share.
 
 ### Example 5: Create a Storage file share with EnabledProtocol proeprty as NFS, and RootSquash property as NoRootSquash
-```
-PS C:\>$share = New-AzRmStorageShare -ResourceGroupName "myresourcegroup" -StorageAccountName "mystorageaccount" -Name "myshare" -EnabledProtocol NFS -RootSquash NoRootSquash 
+```powershell
+$share = New-AzRmStorageShare -ResourceGroupName "myresourcegroup" -StorageAccountName "mystorageaccount" -Name "myshare" -EnabledProtocol NFS -RootSquash NoRootSquash 
 
-PS C:\> $share
```

Comment:
Why remove "PS C:\>" ?
Remove it will make the sample can't differ, is the line a command for input, or an output of a command.

Reply:
Hi @blueww , we are unifying example template. Current example template is as following:

![image](https://user-images.githubusercontent.com/19869716/156715844-9e7bc246-da4c-4ade-b487-1ab948612f1d.png)

The reason why we want to remove `PS C:>` is that we want to allow user to run commands copied from examples directly.
![image](https://user-images.githubusercontent.com/19869716/156716092-6b9da9cc-0316-4885-b1d8-51e753a6eaa6.png)

Users need to remove `PS C:>` before command when they run commands copied from examples. We consider it as an additional action.

Reply:
Thanks @BethanyZhou  for the explaination!
However, I only see  "PS C:>" , but not see command and output separation. 
If only remove "PS C:>", it might make customer can't differ command and output.

BTW, some of the existing examples has several command, and each command has it's own output. How will you handle them？

Reply:
Yes, there is a limit from docs team. Somehow, they will merge code and output to show together. We already reported this issue to docs team. In our side, we need adjust format asap.

> BTW, some of the existing examples has several command, and each command has it's own output. How will you handle them？

We will keep last output, is it true? @CaptainFanZzz

Reply:
For this kind of examples, we defer to change them and keep them as they are.